src/ray/gcs/gcs_server/gcs_node_manager.cc

```diff
@@ -282,6 +283,31 @@ void GcsNodeManager::HandleDeleteResources(const rpc::DeleteResourcesRequest &re
   }
 }
 
+void GcsNodeManager::HandleSetInternalConfig(const rpc::SetInternalConfigRequest &request,
+                                             rpc::SetInternalConfigReply *reply,
+                                             rpc::SendReplyCallback send_reply_callback) {
+  auto on_done = [reply, send_reply_callback](const Status status) {
+    GCS_RPC_SEND_REPLY(send_reply_callback, reply, status);
+  };
+  RAY_CHECK_OK(gcs_table_storage_->InternalConfigTable().Put(UniqueID::Nil(),
```

Comment:
I assume the `UniqueID::Nil()` here is just a workaround to use a known key?

Reply:
Yes it is.